scalameta/parsers/shared/src/main/scala/scala/meta/internal/parsers/ScalametaParser.scala

```diff
@@ -499,7 +499,10 @@ class ScalametaParser(input: Input)(implicit dialect: Dialect) { parser =>
           (nextRegions, currRef)
         } else if (curr.is[EOF]) {
           sepRegions match {
-            case x :: xs if x.isIndented => (xs, mkOutdent(currPos))
+            case x :: xs if x.isIndented =>
+              // do no include trailing comment/whitespace tokens
+              val pointPos = if (currPos - prevPos == 1) currPos else prevPos + 1
+              (xs, mkOutdent(pointPos))
```

Comment:
This fixes what @kitbellew reported in [that comment](https://github.com/scalameta/scalameta/pull/2426#issuecomment-889733564)